libcontainer/configs/mountpropagation_linux.go

```diff
@@ -0,0 +1,9 @@
+// +build linux
+
+package configs
+
+const (
+	MNT_RPRIVATE MountPropagationMode = "rprivate"
```

Comment:
Hmm...., actually it probably does not have to be string. I will look into modifying it.